suites/upgrade/client-upgrade/hammer-client-x/basic/1-install/hammer-client-x.yaml

```diff
@@ -5,6 +5,5 @@ tasks:
 upgrade_workload:
   sequential:
   - install.upgrade:
-      exclude_packages: ['ceph-test', 'ceph-test-dbg']
```

Comment:
@dachary if what we need is just "ceph-coverage" which is just a shell script, can we dispense with ceph-test-dbg?